frontend/app/tab/tab.scss

```
@@ -129,6 +171,18 @@ body:not(.nohover) .tab.dragging {
         border-color: transparent;
         background: rgb(from var(--main-text-color) r g b / 0.1);
     }
+    &.has-indicator .tab-inner {
+        background: rgb(from var(--tab-indicator-color) r g b / 0.15);
```

Stylelint (17.2.0)

[error] Expected "0.15" to be "15%" (alpha-value-notation)

(alpha-value-notation)